### PR TITLE
Training - Adding Angular super power  

### LINKED ISSUE #594: 📄 Events - Angular Superpowers
https://www.ssw.com.au/ssw/Events/Training/Angular-Superpowers-Tour.aspx 

move to `/training/angular-superpowers`

Sidebar - move to a row
Video - move to the carousel
Migrate all text content
Build the booking component

Image

Ask @camillars if we need the Twitter component

Ask @camillars if we need the locations component

Image

Ask @camillars if we need the past events section
Image

```[tasklist]
### Tasks
- [ ] https://github.com/SSWConsulting/SSW.Website/issues/918
- [ ] https://github.com/SSWConsulting/SSW.Website/issues/943
- [ ] https://github.com/SSWConsulting/SSW.Website/issues/964
- [ ] https://github.com/SSWConsulting/SSW.Website/issues/1003
- [ ] https://github.com/SSWConsulting/SSW.Website/issues/1022
- [x] Mega Menu - Update route
```
- [ ] ~Front Door - Add redirect~

### PR DESCRIPTION
**Pull request by amankumarrr**
related to #594 

Affected routes:

- `/training/angular-superpowers-tour` 

**Screenshots:**

image

**Figure: Angular super power page - 1** 

image

**Figure: Angular super power page - 2** 

image

**Figure: Angular super power page - 3** 

image

**Figure: Angular super power page - 4**

### CODE REVIEW

File: components/training/presenterBlock.tsx [hunk 1]
```diff
@@ -55,7 +55,8 @@ export const PresenterBlock = ({ data }) => {
 const PresenterCard = ({ presenter, schema, index }) => {
   return (
     <div
-      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl md:h-full"
+      // eslint-disable-next-line tailwindcss/no-arbitrary-value
+      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl md:h-[680px] md:w-96"
```
Comment (bradystroud) [inline components/training/presenterBlock.tsx]: Move the arbitraty height to tailwind config

**Review by bradystroud — CHANGES_REQUESTED** [review]
lgtm

File: components/training/presenterBlock.tsx (hunk 1 above)
Comment (amankumarrr) [reply components/training/presenterBlock.tsx]: done

**Review by bradystroud — APPROVED** [review]

**Comment by wicksipedia** [comment]
@amankumarrr pro tip for next time, get a full page screenshot
https://www.microsoft.com/en-us/edge/features/web-capture?form=MT00D8

**Pull request merged**